data/pokemon_upgrade_cost.json

```diff
@@ -0,0 +1,82 @@
+[
+[1, 200],
+[1, 200],
```

Comment:
Index in the list is `pokemon.level * 2 - 1`. First value is candy count, second is stardust.